### PR TITLE
Update dependencies

### LINKED ISSUE #1259: Do not require git for installation
### Describe the problem and steps to reproduce it:

Try to install the linter (or `web-ext`) without having git on your machine

### What happened?

You get a cryptic `npm` error saying that git is not installed, probably triggered by this dependency: https://github.com/mozilla/addons-linter/blob/ca8b718821523fd9ad80dc8cbdd0018f49c0bba4/package.json#L82

### What did you expect to happen?

You should be able to install the linter without git

### Anything else we should know?
(Please include screenshots and any relevant files.)

@EnTeQuAk could you publish that module to npm instead?

Here is the web-ext bug report https://github.com/mozilla/web-ext/issues/933

### PR DESCRIPTION
**Pull request by wagnerand**
Fixes #1259

### CODE REVIEW

**Comment by wagnerand** [comment]
Test failure on node4 only :( but we're dropping them in #1272.

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "dispensary": "0.10.5",
     "es6-promisify": "5.0.0",
     "eslint": "3.19.0",
-    "eslint-plugin-no-unsafe-innerhtml": "EnTeQuAk/eslint-plugin-no-unsafe-innerhtml#4b6b606d50",
+    "eslint-plugin-no-unsafe-innerhtml": "1.0.16",
```
Comment (muffinresearch) [inline package.json]: This is now called `eslint-plugin-no-unsanitized` but it could have breaking changes so it would make sense to handle it separately.

Comment (muffinresearch) [inline package.json]: I filed https://github.com/mozilla/addons-linter/issues/1273 for that

Comment (wagnerand) [inline package.json]: Thank you!

**Comment by wagnerand** [comment]
Tests pass, assuming r+. 😄

**Pull request merged**